<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>calendar</class>
 <widget class="QMainWindow" name="calendar">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1207</width>
    <height>731</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>calendar</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>581</width>
      <height>101</height>
     </rect>
    </property>
    <widget class="QLabel" name="time_label">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>10</y>
       <width>541</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="date_label">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>60</y>
       <width>541</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>180</y>
      <width>535</width>
      <height>460</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout"/>
   </widget>
   <widget class="QLabel" name="selected_date_label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>130</y>
      <width>221</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pre_month_pushButton">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>130</y>
      <width>60</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>上一月</string>
    </property>
   </widget>
   <widget class="QPushButton" name="next_month_pushButton">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>130</y>
      <width>60</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>下一月</string>
    </property>
   </widget>
   <widget class="QPushButton" name="add_plan_pushbutton">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>130</y>
      <width>80</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>添加日程</string>
    </property>
   </widget>
   <widget class="QPushButton" name="all_plans">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>130</y>
      <width>80</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>全部日程</string>
    </property>
   </widget>
   <widget class="QPushButton" name="search_day_Button">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>130</y>
      <width>80</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>查找日期</string>
    </property>
   </widget>
   <widget class="QTableView" name="day_plans">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>10</y>
      <width>581</width>
      <height>631</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="delete_Button">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>610</y>
      <width>581</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>删除该日程</string>
    </property>
   </widget>
   <widget class="QPushButton" name="refresh_Button">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>580</y>
      <width>581</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>刷新</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1207</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
